tensorflow/tf_version_script.lds

```diff
@@ -2,6 +2,7 @@ tensorflow {
   global:
     *tensorflow*;
     *perftools*gputools*;
+    TF_*;
```

Comment:
This appears to be case sensitive, and the C API functions are all prefixed with `TF_`.